selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/inject/SeleniumTestHandler.java

```diff
@@ -93,6 +94,10 @@
   @Named("tests.htmldumps_dir")
   private String htmldumpsDir;
 
+  @Inject
+  @Named("tests.webDriverLogsDir")
```

Comment:
use the same test properties naming pattern, please: `tests.webDriverLogsDir` > `tests.webdriverlogs_dir`